definitions/detect_variants.wdl

```diff
@@ -42,6 +42,8 @@ workflow detectVariants {
     Float varscan_p_value = 0.99
     Float? varscan_max_normal_freq
 
+    Float? min_var_freq = varscan_min_var_freq
```

Comment:
I think the way this PR is setup right now, if someone ran immuno.wdl then min_var_freq would get set to `0.05` by default. And if someone ran somatic_exome.wdl then min_var_freq would be set to `0.05` via varscan_min_var_freq.  But if someone were to run detect_variants.wdl directly, then min_var_freq would be set to `0.05` via varscan_min_var_freq. I think this is confusing.

Reply:
I will assert that best practice should be to set the default value for a tool in the tool itself, and then only expose it upstream (or override it) when necessary because different users might want it set differently.  

This seems like a case where it could be set as 0.05 on the tool, then exposed (with no default) upstream for folks that would like to change it. (I do know that people are modifying this one sometimes for low-VAF calling)